pkg/cluster/config/types.go

```diff
@@ -25,58 +25,54 @@ import (
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// Config groups all nodes in the `kind` Config.
-type Config struct {
+// Cluster contains kind cluster configuration
+type Cluster struct {
 	// TypeMeta representing the type of the object and its API schema version.
-	metav1.TypeMeta
+	metav1.TypeMeta `json:",inline"`
 
-	// Nodes contains the list of nodes defined in the `kind` Config
-	Nodes []Node `json:"nodes,"`
-}
+	// Nodes contains the list of nodes defined in the `kind` Cluster
+	Nodes []Node `json:"nodes"`
```

Comment:
Internal API types shouldn't have JSON tags. They will never actually be serialised/deserialised, and historically certain generators have tripped up on this (they've used the presence of even a single JSON tag to denote the API types as 'external')

Reply:
Interesting. Will drop these.

Reply:
yep, i missed that.

Reply:
done